File: ecommerce_api/src/main/java/ecommerce_api/ecommerce_api/dto/SancionView.java
package ecommerce_api.ecommerce_api.dto;

import java.time.LocalDate;

/**
 * The type Sancion view.
 */
public record SancionView(
        Long id,
        Long usuarioId,
        String usuarioNombre,
        Long moderadorId,
        String moderadorNombre,
        String motivo,
        LocalDate fecha,
        String estado
) { }

File: ecommerce-app/src/app/entities/SancionView.ts
export interface SancionView {
  id: number;
  usuarioId: number;
  usuarioNombre: string;
  moderadorId: number;
  moderadorNombre: string;
  motivo: string;
  fecha: string;
  estado: 'ACTIVA'|'LEVANTADA';
}
